assets/src/scripts/components/hydrograph/parameters.js

```diff
@@ -181,7 +181,7 @@ export const plotSeriesSelectTable = function (elem,
             .attr('aria-selected', parm => parm[1].selected)
             .on('click', function (parm) {
                 if (!parm[1].selected) {
-                    store.dispatch(Actions.updateCurrentVariable(siteno, parm[1].variableID));
+                    store.dispatch(Actions.updateIVCurrentVariableAndRetrieveTimeSeries(siteno, parm[1].variableID));
```

Comment:
I notice the word `and` in this action.  Is it possible to split this action into two actions, one to update the current variable and one to retrieve the time series?  Maybe it's better they are paired.

Reply:
We could, however this two things need to be paired. There is a separate action to set the current variable but I wanted to be clear that this Action not only updated the state but also fetches data.